adcircpy/fort15.py

```diff
@@ -582,15 +623,50 @@ def NSCREEN(self):
             return 100
 
     @property
-    def NWS(self):
+    def NWS(self) -> int:
+        """
+        wind stress number
+        http://adcirc.org/home/documentation/users-manual-v50/input-file-descriptions/nws-values-table/
+        """
+
         if self._runtype == 'coldstart':
-            return 0
-        else:
-            if self.wind_forcing is not None:
+            nws = 0
+        elif self.mesh is not None:
+            wind_forcing = self.mesh._surface_forcing['imetype']
+            if wind_forcing is not None:
                 # check for wave forcing here as well.
-                return self.wind_forcing.NWS
+                nws = int(wind_forcing.NWS % 100)
+        else:
+            nws = 0
+
+        return nws
+
+    @property
+    def NRS(self) -> int:
+        """
+        radiative (wave) stress number
+
+        100 - fort.23
+        300 - SWAN
+        400 - STWAVE
+        500 - WW3
+        """
+
+        if self._runtype == 'coldstart':
+            nrs = 0
+        elif self.mesh is not None:
+            wind_forcing = self.mesh._surface_forcing['imetype']
+            wave_forcing = self.mesh._surface_forcing['iwrtype']
+            if wave_forcing is not None:
+                nrs = wave_forcing.NRS
+            elif wind_forcing is not None:
+                nrs = int(math.floor(wind_forcing.NWS / 100))
             else:
-                return 0
+                nrs = 0
+        else:
+            nrs = 0
+
+        return nrs
```

Comment:
NRS parameter extracted from wave forcing class OR from hundredths place of wind forcing NWS configuration